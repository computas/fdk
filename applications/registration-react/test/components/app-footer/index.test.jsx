import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../../src/components/app-footer';

test('should render Footer correctly', () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
});
