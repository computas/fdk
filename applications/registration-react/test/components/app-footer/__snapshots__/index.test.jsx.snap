// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer correctly 1`] = `
<footer
  className="fdk-footer"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-3"
      >
        <p
          className="fdk-p-footer"
        >
          <a
            href="https://www.brreg.no/personvernerklaering/"
          >
            Informasjonskapsler
            <br />
             og personvern
            <i
              className="fa fa-external-link fdk-fa-right"
            />
          </a>
        </p>
      </div>
      <div
        className="col-md-6 text-center"
      >
        <span
          aria-hidden="false"
          className="uu-invisible"
        >
          Felles Datakatalog.
        </span>
        <p
          className="fdk-p-footer"
        >
          Dette er et samarbeidsprosjekt mellom Brønnøysundregistrene, Difi og øvrige SKATE-etater.
        </p>
      </div>
      <div
        className="col-md-3 text-right"
      >
        <p
          className="fdk-p-footer"
        >
          <a
            href="mailto:fellesdatakatalog@brreg.no"
          >
            <span
              aria-hidden="false"
              className="uu-invisible"
            >
              Mailadresse.
            </span>
            Kontakt:
            <br />
            fellesdatakatalog@brreg.no
          </a>
        </p>
      </div>
    </div>
  </div>
</footer>
`;
